scripts/check-and-fix.sh

```diff
@@ -4,17 +4,16 @@ git config --global user.email "travis@travis-ci.org"
 git config --global user.name "Travis CI"
 git checkout $TRAVIS_BRANCH
 
-node_version=$(node --version | cut -d. -f1 | cut -c2-)
-
 echo ""
 echo "------- Checking Schema -------"
 echo ""
 
 # Commit the schema if outdated
+yarn schema
```

Comment:
Ah I got an idea!

Since schema takes a minute, I think it's better to split this file into  

1) Update schema
2) Fix Lint
3) Update Example

1 and 2 could go into the "build" VM (since build VM has to run lint and fix schema anyway so we better run lint fix before linting and leverage the schema built in the VM.

The "fix" VN could then only become "update example", which already takes time on its own especially when SVG changes.

Reply:
Isn't the problem that we might push a change from the update schema VM and the update example VM that both trigger a build then?

Reply:
Fair.   Then it's actually more time to restart the Travis Builds. (not worth the schema building time anymore)

Reply:
Somehow the builds are really slow in this PR anyway. Not sure why.